bin/ch-builder2tar

```diff
@@ -128,6 +128,32 @@ EOF
     tar rf "$tar" -b1 -P --xform="s|${temp}|ch/environment|" "$temp"
     rm "$temp"
 
+    echo "adding metadata"
+    temp=$(mktemp --tmpdir ch-builder2tar.XXXXXX)
+    buildah inspect --format='{{print "{\n"}}'\
+'{{printf "  \"arch\": %q,\n" .Docker.Architecture}}'\
+'{{print "  \"cwd\": "}}{{if .Docker.Config.WorkingDir}}'\
+'{{.Docker.Config.WorkingDir}}{{else}}{{print "\"\""}}{{end}}{{print ",\n"}}'\
```

Comment:
Wouldn't it be better to omit the `cwd` key if `.Docker.Config.WorkingDir` is not set?

Reply:
I was imitating the way that our current metadata works in ch-image. If a parameter does not exist, we place the empty set of whatever data type it is `"", [], {}`.

Reply:
We can definitely remove this if it doesn't exist instead.